.github/workflows/ISSUE_TEMPLATE/bug-report.yaml

```
@@ -0,0 +1,53 @@
+name: 🐞 Bug
+description: File a bug/issue
+labels: [Bug, Needs Triage]
+body:
+  - type: textarea
+    id: problem
+    attributes:
+      label: What happened?
+      description: |
+        Please provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner.
+        If this matter is security related, please disclose it privately via https://discord.gg/hyperledger
```

Regarding security related issues, we should probably ask for people to send an email to security@hyperledger.org. There is currently no place to privately report security issues on Discord.
```suggestion
        If this matter is security related, please disclose it privately via security@hyperledger.org
```